packages/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+    "files": [],
+    "references": [{ "path": "near-api-js" }]
```

Comment:
What's this for? Is it so typescript can resolve imports across different packages (typesciprt projects)? If so, I don't think this is the right way to go about it. Yarn workspaces/Lerna will (sym)link packages together within `node_modules` which allows us to reference other packages in the monorepo.

Reply:
Was initially using to build entire monorepo from root, can be removed now